packages/application-generic/src/health/cache.health-indicator.ts

```diff
@@ -3,34 +3,29 @@ import {
   HealthIndicator,
   HealthIndicatorResult,
 } from '@nestjs/terminus';
-import { Injectable, Logger } from '@nestjs/common';
-
+import { Injectable } from '@nestjs/common';
 import { CacheService } from '../services/cache';
 
-const LOG_CONTEXT = 'CacheServiceHealthIndicator';
-
 @Injectable()
 export class CacheServiceHealthIndicator extends HealthIndicator {
-  private INDICATOR_KEY = 'cacheService';
+  private static KEY = 'cacheService';
 
   constructor(private cacheService: CacheService) {
     super();
   }
 
   async isHealthy(): Promise<HealthIndicatorResult> {
-    const isReady = this.cacheService.cacheEnabled();
+    const isHealthy = this.cacheService.cacheEnabled();
+    const result = this.getStatus(CacheServiceHealthIndicator.KEY, true);
```

Comment:
Should not we pass here isHealthy?

```suggestion
    const result = this.getStatus(CacheServiceHealthIndicator.KEY, isHealthy);
```

Reply:
Great catch!